packages/react-icons/package.json

```
@@ -40,7 +40,8 @@
     "yargs": "^14.0.0"
   },
   "peerDependencies": {
-    "@griffel/react": "^1.0.0"
+    "@griffel/react": "^1.0.0",
+    "react": "^17.0.1"
```

This should match Fluent's range:

```suggestion
    "react": ">=16.8.0 <18.0.0",
```

https://github.com/microsoft/fluentui/blob/a99218c3e02097545901e6a56cd6fe8aaae83661/packages/react-avatar/package.json#L56